### ISSUE TITLE
Extention not work and throw error when clicked

### ISSUE THREAD

**Issue by surya27895** [kyrie25/spicetify-queue-panel#3]
here my spotify and spicetify ver
```
Spotify for Windows (32 bit)
1.2.17.832.g15fa351e
Spicetify v2.22.1
Theme: marketplace
Extensions: shuffle+.js, popupLyrics.js, css-editor.js, romaja_lyrics.js, romaji_lyrics.js, fullAlbumDate.js, romaji-convert.js, share-on-twitter.js, noTrackingLinks.js, tracksToEdges.js, AnonymizedRadios.js, fullScreenWrapper.js, genre.js, utilities.js, copytoclipboard2.js, adblock.js, queuePanel.js
Custom apps: marketplace, lyrics-plus
```
and here some error screenshot
![image](https://github.com/kyrie25/spicetify-queue-panel/assets/22859567/84bbc074-afb8-4dc3-ab08-3df10d1584cf)
![image](https://github.com/kyrie25/spicetify-queue-panel/assets/22859567/d9ba623f-244b-4f95-911d-452f8b69eb29)
![image](https://github.com/kyrie25/spicetify-queue-panel/assets/22859567/6df5adea-1ee2-452d-8ecb-1a702327f6e6)

**Comment by kyrie25** [comment]
This is an error on the CLI side, wait for a future Spicetify release.
https://github.com/spicetify/spicetify-cli/pull/2496

**Comment by kyrie25** [comment]
Temp fix in f505d7e4655f4229381da117471341e6db536c50
